flink-table/flink-table-api-scala/src/test/scala/org/apache/flink/table/types/extraction/DataTypeExtractorScalaTest.scala

```diff
@@ -37,8 +37,8 @@ class DataTypeExtractorScalaTest {
   def testScalaExtraction(testSpec: DataTypeExtractorTest.TestSpec): Unit = {
     if (testSpec.hasErrorMessage) {
       assertThatThrownBy(() => runExtraction(testSpec))
-        .isInstanceOf(classOf[ValidationException])
         .is(HamcrestCondition.matching(errorMatcher(testSpec)))
+        .isInstanceOf[ValidationException]
```

Comment:
if there a reason to change the order of these tests with this change?

Reply:
yes I read the description, there are 2 changes 
.isInstanceOf(classOf[ValidationException]) -> .isInstanceOf[ValidationException]
and changing the order so the check is now last

 I am just checking whether we need to change the order. It is a nit, so not a big deal.